File: fixtures/scale/objc/large/Families/Bacon/BaconVectors.m
#import "BaconVectors.h"
#import "CLTestVector.h"

@implementation BaconVectors

+ (NSArray *)all {
    NSMutableArray *vectors = [NSMutableArray array];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"ATTACKATDAWN" expected:@"EYZHKTKEPNKC"]];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"THEQUICKBROWNFOX" expected:@"XMKXCRMVNECLDWGQ"]];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"DEFENDTHEEASTWALL" expected:@"HJLLVMDSQROHJNSEF"]];
    return vectors;
}

@end

File: fixtures/scale/swift/large/Families/Bacon/BaconVectors.swift
// Known-answer test data captured from BaconCipher's own encode().
enum BaconVectors {
    static func all() -> [TestVector] {
        return [
            TestVector(plaintext: "ATTACKATDAWN", expected: "EYZHKTKEPNKC"),
            TestVector(plaintext: "THEQUICKBROWNFOX", expected: "XMKXCRMVNECLDWGQ"),
            TestVector(plaintext: "DEFENDTHEEASTWALL", expected: "HJLLVMDSQROHJNSEF"),
        ]
    }
}
